.github/workflows/test-ci-command.yml

```diff
@@ -34,7 +34,7 @@ jobs:
         && (!contains(github.event.issue.labels.*.name, 'safe-to-test') || github.event.issue.user.name != github.event.comment.user.name)
       run: exit 1
     - name: React to comment
-      uses: actions/github-script@v4
+      uses: actions/github-script@v6
```

Comment:
v6/v5 has some breaking changes... I think they no longer have the `github.reactions.createForIssueComment` option

Reply:
yeah seen it.